### PR TITLE
Allowing specification of container manager

### PR DESCRIPTION
**Pull request by dkosteck**
Updating documentation
Updating test files for container_manager field in config Updating testdata specification for container commands NOTE: Some tests still specify their own container commands if they differ heavily from the testdata specified ones, or are only used in a singular test
Updating calls to generic container commands
Formatting

Combined report: https://gist.github.com/dkosteck/520e13f41c9277f72937e19a6f71bd4a
Note: IPv6 test failing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Two ports of the NIC under test are directly connected to the traffic generator
 On the DUT server, the following RPM packages are required,
 * tmux
 * nmap
-* podman
+* a container manager (podman or docker)
```
Comment (jianzzha) [inline README.md:19]: Maybe put a recommendation to use podman  for RHEL and how to install podman if it is missing, e.g. 'yum -y install podman'. For docker it is out of the documentation scope.

**Comment by jianzzha** [comment]
by the way, the ipv6 case failure should be cleared after ipv6 permutation test PR is merged.

File: README.md (hunk 1 above)
Comment (dkosteck) [reply README.md]: Added this to the README in the most recent commit

**Review by jianzzha — APPROVED** [review]
Looks good

**Comment by jianzzha** [comment]
@dkosteck Please resolve the conflict so that I can merge it

**Pull request merged**